src/dc_location.rs

```diff
@@ -13,7 +13,7 @@ use crate::types::*;
 use crate::x::*;
 
 // location handling
-#[derive(Copy, Clone)]
+#[derive(Clone, Default)]
 #[repr(C)]
 pub struct _dc_location {
```

Comment:
please remove the `_` in front as part of this

Reply:
I am going to close this PR and open another one where renaming is a first commit.